src/robot.js

```diff
@@ -232,24 +220,24 @@ class Robot {
     })
   }
 
-  // Public: Registers new middleware for execution after matching but before
-  // Listener callbacks
-  //
-  // middleware - A function that determines whether or not a given matching
-  //              Listener should be executed. The function is called with
-  //              (context, next, done). If execution should
-  //              continue (next middleware, Listener callback), the middleware
-  //              should call the 'next' function with 'done' as an argument.
-  //              If not, the middleware should call the 'done' function with
-  //              no arguments.
+  // Public: Registers new middleware for execution after matching but before listener callbacks
   //
+  // middleware:
+    /*            
+                  A function that determines whether or not a given matching
+                  Listener should be executed. The function is called with
+                  (context, next, done). If execution should
+                  continue (next middleware, Listener callback), the middleware
+                  should call the 'next' function with 'done' as an argument.
+                  If not, the middleware should call the 'done' function with
+                  no arguments.
+    */
```

Comment:
I’m not sure about this change, I would leave the `//` comments. The other changes are great :)